### PR TITLE
KEYCLOAK-6494: Address load time of new acct mgt console

### PR DESCRIPTION
**Pull request by ssilvert**
This also takes care of:
KEYCLOAK-6495: Address number of requests for new acct mgt console
KEYCLOAK-6496: Cleanup and polish current code base for new acct mgt console

### CODE REVIEW

File: services/src/main/java/org/keycloak/services/resources/account/AccountConsole.java [hunk 1]
```diff
@@ -101,7 +102,9 @@ public Response getMainPage() throws URISyntaxException, IOException, FreeMarker
                 if (auth != null) {
                     Locale locale = session.getContext().resolveLocale(auth.getUser());
                     map.put("locale", locale.toLanguageTag());
-                    map.put("msg", messagesToJsonString(theme.getMessages(locale)));
+                    Properties messages = theme.getMessages(locale);
+                    map.put("javaMessages", new MessageFormatterMethod(locale, messages));
```
Comment (stianst) [inline services/src/main/java/org/keycloak/services/resources/account/AccountConsole.java]: Where is javaMessages used? I couldn't see it being used anywhere. In either case I would use a name not including Java. Perhaps "advancedMsg"?

Comment (stianst) [inline services/src/main/java/org/keycloak/services/resources/account/AccountConsole.java]: Where is javaMessages used? I couldn't see it being used anywhere. In either case I would use a name not including Java. Perhaps "advancedMsg"?

File: themes/src/main/resources/theme/keycloak-preview/account/index.ftl [hunk 2]
```diff
@@ -65,56 +58,162 @@
...
-            </#list>
-        </#if>
-
-        <script src="${resourceUrl}/node_modules/zone.js/dist/zone.js"></script>
-        <script src="${resourceUrl}/node_modules/systemjs/dist/system.src.js"></script>
-
-        <script src="${resourceUrl}/systemjs.config.js"></script>
+        <script src="${authUrl}/js/keycloak.js"></script>
+
+<!--
```
Comment (stianst) [inline themes/src/main/resources/theme/keycloak-preview/account/index.ftl]: We shouldn't have comments in the templates/html. For any work that needs to be done there should be JIRAs for it, not comments.

File: themes/src/main/resources/theme/keycloak-preview/account/index.ftl [hunk 3]
```diff
@@ -65,56 +58,162 @@
...
-            }
 
-            .kc-logo-text {
-                background-image: url("${resourceUrl}/img/keycloak-logo-text.png");
-                background-repeat: no-repeat;
-                width: 250px;
-                height: 38px;
-            }
-        </style>
+   <!-- TODO: We should save these css and js into variables and then load in
```
Comment (stianst) [inline themes/src/main/resources/theme/keycloak-preview/account/index.ftl]: See above

File: themes/src/main/resources/theme/keycloak-preview/account/index.ftl [hunk 4]
```diff
@@ -65,56 +58,162 @@
...
+            <div class="navbar-header">
+                <a href="http://www.keycloak.org" class="navbar-brand">
+                    <img class="navbar-brand-icon" type="image/svg+xml" src="${resourceUrl}/app/assets/img/keycloak-logo-min.png" alt="" width="auto" height="30px"/>
+                </a>
+            </div>
+            <nav class="collapse navbar-collapse">
+                <ul class="nav navbar-nav">
+                </ul>
+                
+                <!-- This sign in button is only displayed in the rare case where we go directly to this page and we aren't logged in.
```
Comment (stianst) [inline themes/src/main/resources/theme/keycloak-preview/account/index.ftl]: See above

**Review by stianst — COMMENTED** [review]
Are the added backgrounds used anywhere?

We need a JIRA to make sure we use the NPM approach used by the rest of the themes rather than having a separate package.json and yarn. At the same time we should get rid of the account2 profile so we can start including the account2 in regular builds/dists. By including the common resources in the theme the node_modules will be available.

I'm not sure what should be working at this stage, but I'm getting a few errors and weird behaviour in the console. It does load quickly though.

File: services/src/main/java/org/keycloak/services/resources/account/AccountConsole.java (hunk 1 above)
Comment (ssilvert) [reply services/src/main/java/org/keycloak/services/resources/account/AccountConsole.java]: javaMessages is not being used right now.  The reference to it in index.ftl was removed in the latest refactoring.  I'll fix.

File: themes/src/main/resources/theme/keycloak-preview/account/index.ftl (hunk 2 above)
Comment (ssilvert) [reply themes/src/main/resources/theme/keycloak-preview/account/index.ftl]: OK.  BTW, we do have a jira for getting rid of yarn in this code.  KEYCLOAK-6977.

File: themes/src/main/resources/theme/keycloak-preview/account/index.ftl (hunk 4 above)
Comment (ssilvert) [reply themes/src/main/resources/theme/keycloak-preview/account/index.ftl]: For now, I want to leave this one in so Tao and June will know what is going on.

**Comment by ssilvert** [comment]
I think those backgrounds are no longer used.  I'll look into getting rid of them.

The only errors I see in the js console are referring to jQuery.  Bootstrap throws these errors when jQuery is not loaded.  I'm not sure if we want to use jQuery or not.  It does "turn on" a few Bootstrap features that we may want.

**Review by stianst — APPROVED** [review]
Approving this. I'll take a deeper look next week, but as it's preview and not enabled by default we don't need to block merging the PR

**Pull request merged**